ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolAdmin/service/ToolAdminServiceFacade.java

```diff
@@ -0,0 +1,26 @@
+package org.iplantc.de.admin.desktop.client.toolAdmin.service;
+
+import org.iplantc.de.client.models.tool.Tool;
+import org.iplantc.de.client.models.tool.ToolImportUpdateRequest;
+import org.iplantc.de.client.models.tool.ToolImportUpdateRequestList;
+
+import com.google.gwt.user.client.rpc.AsyncCallback;
+
+import java.util.List;
+
+/**
+ * Created by aramsey on 10/28/15.
+ */
+public interface ToolAdminServiceFacade {
+
+    void getTools(String searchTerm, AsyncCallback<List<Tool>> callback);
+
+    void getToolDetails(String toolId, AsyncCallback<ToolImportUpdateRequest> callback);
```

Comment:
Since the new `ToolImportUpdateRequest` model is used to load tool details in addition to making import/update requests, and it already shares all the fields of the existing `Tool` model, we can add all the new required fields to the existing `Tool` model and use it everywhere this new `ToolImportUpdateRequest` model is used instead.